.vscode/settings.json

```diff
@@ -1,3 +1,18 @@
 {
-  "cSpell.words": ["Dootstrapper", "bootstrapper"]
+  "cSpell.words": [
+    "Cloudformation",
+    "Codebuild",
+    "Codepipeline",
```

Comment:
Some of these should be camel case

Reply:
these names are coming from aws-cdk as `aws-cdk/codepipeline` and `aws-cdk/codebuild` and also cloudformation referes to these as `codepipeline` and `codebuild` single words, so leaving it as is.

Reply:
Ah, the module names themselves - I get it.